prep-css-flexbox/starwars.css

```diff
@@ -0,0 +1,15 @@
+/*
+
+.star-wars-card {
+  padding: 0.5rem;
+  border: 1px solid lightgray;
+  border-radius: 0.25rem;
+  box-shadow: 0 8px 8px -8px rgb(0 0 0 / 50%);
+}
+
+.star-wars-card img {
+  width: 100%;
+  background-color: lightgray;
+}
+
+*/
```

Comment:
If you uncomment this you won't have to copy-paste it into the other files.

Reply:
![image](https://user-images.githubusercontent.com/99709530/155857419-881812a0-bcd9-420c-a7c2-6aedc7ab07f2.png)